ui/pages/settings/experimental-tab/__snapshots__/experimental-tab.test.js.snap

```diff
@@ -178,91 +178,6 @@ exports[`ExperimentalTab with desktop enabled renders ExperimentalTab component
         </div>
       </div>
     </div>
-    <h4
```

Comment:
Could we mock the timer in this test so it doesn't blow up in November?

Reply:
Solved in https://github.com/MetaMask/metamask-extension/pull/21221/commits/eef5ff1a2dfd01eaafe986769ee35872c389b4cb